docs/swagger.yml

```diff
@@ -136,6 +136,27 @@ paths:
           description: "Host credential deleted"
         404:
           description: "Host credential not found"
+  /reports/:
+    get:
+      tags:
+        - "Report"
+      summary: "Retrieve list of reports for systems scanned"
+      description: "List reports for systems that have been scanned."
+      operationId: "listReports"
+      produces:
+      - "application/json"
+      parameters:
+        - name: "fact_collection_id"
+          in: "query"
+          description: "The Fact Collection ID to be queried"
+          type: "string"
+      responses:
+        200:
+          description: "List of reports for scanned systems"
+          schema:
+            type: "array"
+            items:
+              $ref: "#/definitions/SystemReportOut"
```

Comment:
potential 404 condition here if the fact_collection_id isn't found

Reply:
Added 404 to swagger